pulp_ansible/app/galaxy/views.py

```diff
@@ -28,12 +28,18 @@ class GalaxyVersionView(views.APIView):
     authentication_classes = []
     permission_classes = []
 
-    def get(self, request, path):
+    v3_only = False
```

Comment:
Is this expected to still be here?

Reply:
I think it's still confused about why it needs to be switchable yet I don't see where `v3_only` is ever set to `True`.

Reply:
@bmbouter https://github.com/pulp/pulp_ansible/pull/849/files#diff-a18275319f35fc1fbea4b63cd3e4a3b8fc68d4a43e51f7702d39818c705d1e2aR212

Reply:
The v1 and v2 APIs all require a distro base path in the url for the entire API, however the v3 API is designed for the base path to be nested further down in URL path. For example a v2/ legacy v3 path would look like `/pulp_ansible/galaxy/{path}/api/v3/collections` whereas the new v3 paths look like `/pulp_ansible/galaxy/default/api/v3/plugin/ansible/content/{path}/`. Since v1 and v2 both require a path at the root, it doesn't make sense to serve them from `/pulp_ansible/galaxy/default/`.